apps/components-e2e/src/components/filter-field/filter-field.e2e.ts

```diff
@@ -76,3 +77,30 @@ test('should remove all filters when clicking the clear-all button', async (test
   await testController.wait(300);
   await testController.expect(await filterTags.exists).notOk();
 });
+
+test('should remove all removable filters when clicking the clear-all button', async (testController: TestController) => {
+  // Setup the second datasource.
+  await testController.click(setupSecondTestScenario);
+
+  // Wait for the filterfield to catch up.
+  await testController.wait(500);
+
+  // Manually set an option, because this seems to break the
+  // clear all change detection.
+  await clickOption(1);
+  await clickOption(1);
+
+  await testController.expect(await filterTags.count).eql(2);
+
+  // // Click somewhere outside so the clear-all button appears
+  await testController.click(Selector('.outside'));
+  // Wait for the filterfield to catch up.
+  await testController.wait(500);
+  await testController.expect(await clearAll.exists).ok();
+
+  // Click the clear all-button, the created filter should be removed
+  await testController.click(clearAll);
+  // Wait for the filterfield to catch up.
+  await testController.wait(500);
```

Comment:
is this related to animations?

Maybe we should consider using the `NoopAnimationsModule` instead of the `BrowseranimationsModule`?
@ffriedl89 Maybe this would reduce flakiness?

Reply:
It's not related to the animations module, but the way the filter field updates, it waits for the `zone` to get stable, before it will update it's state.